mendix/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # CHANGELOG - Mendix
 
+## 1.0.1 / 2023-04-19
+* [UPDATED] Dashboard now supports filtering by env
+
```

Comment:
@pappachino This can be removed since this is only published in your sandbox and not published live. Once this is done, we can publish to your sandbox!